keyman/KeyClient.py

```diff
@@ -29,8 +29,10 @@ def __init__(self, platform: str, logger: Logger):
         if not platform:
             raise ValueError("Platform must not be empty")
         
-        self.stdout = conn
-        self.stdin: PubSub = conn.pubsub(ignore_subscribe_messages = True)
+        self.conn = get_redis_connection()
```

Comment:
The change here is moving the refernce from the library to this call to the `get_redis_connection()` method.